src/toolkit/timeseries.cc

```diff
@@ -0,0 +1,13 @@
+#include "timeseries.h"
+
+namespace cyclus {
+namespace toolkit {
+
+template <>
+void RecordTimeSeries<POWER>(cyclus::Agent* agent, double value) {
+  RecordTimeSeries<double>("Power", agent, value);
```

Comment:
Not a stopper, but FYI - you could just `RecordTimeSeries("Power", agent, value);` and the compiler will infer the templated type for the function.